src/macros.rs

```diff
@@ -15,6 +15,12 @@ macro_rules! assert_buffer {
     }};
 }
 
+macro_rules! assert_buffer_eq {
+    ($length:expr, $target:expr, $name:expr) => {{
+        assert_buffer!($length, $target, $length == $target, $name)
```

Comment:
This line asserts something about buffer lengths not buffers themselves. Would make sense to call it `assert_buffer_len!`.